assets/scripts/segments/drag_and_drop.js

```diff
@@ -1,55 +1,40 @@
 import { trackEvent } from '../app/event_tracking'
 import { loseAnyFocus } from '../util/focus'
-import { getStreetSectionTop } from '../app/window_resize'
 import { infoBubble } from '../info_bubble/info_bubble'
 import { app } from '../preinit/app_settings'
-import { system } from '../preinit/system_capabilities'
 import { setIgnoreStreetChanges } from '../streets/data_model'
 import { getElAbsolutePos } from '../util/helpers'
 import { generateRandSeed } from '../util/random'
-import { BUILDING_SPACE } from './buildings'
 import {
   SegmentTypes,
   getSegmentInfo,
   getSegmentVariantInfo
 } from './info'
 import {
-  SHORT_DELAY,
   RESIZE_TYPE_INITIAL,
   RESIZE_TYPE_DRAGGING,
   RESIZE_TYPE_PRECISE_DRAGGING,
   MIN_SEGMENT_WIDTH,
   resizeSegment,
   handleSegmentResizeEnd,
   normalizeSegmentWidth,
-  scheduleControlsFadeout,
   cancelFadeoutControls,
   hideControls,
   cancelSegmentResizeTransitions
 } from './resizing'
 import { getVariantArray, getVariantString } from './variant_utils'
-import { TILE_SIZE } from './constants'
-import {
-  setSegmentContents,
```

Comment:
This function is no longer needed at all -- could it be deleted?

Reply:
Ah, yea I seem to have missed a lot of things that are no longer needed. My bad.